<?xml version="1.0"?>
<launch>
  <arg name="csv_path_accel_map" default="$(find-pkg-share aichallenge_submit_launch)/data/accel_map.csv"/>
  <arg name="csv_path_brake_map" default="$(find-pkg-share aichallenge_submit_launch)/data/brake_map.csv"/>

  <node pkg="aichallenge_awsim_adapter" exec="actuation_cmd_converter">
    <param name="csv_path_accel_map" value="$(var csv_path_accel_map)"/>
    <param name="csv_path_brake_map" value="$(var csv_path_brake_map)"/>
    <param name="max_steering_rotation_rate" value="0.5"/>
    <param name="steering_angle_delay" value="0.065"/>
  </node>
  <node pkg="aichallenge_awsim_adapter" exec="sensor_converter">
    <param from="$(find-pkg-share aichallenge_awsim_adapter)/config/sensor_converter.param.yaml" />
  </node>
  <node pkg="aichallenge_awsim_adapter" exec="make_disturbance" output="screen" prefix="xterm -e">
    <param name="steering_disturbance" value="0.1"/>
    <param name="accleration_disturbance" value="0.1"/>
    <prefix>xterm -e</prefix>
  </node>
</launch>
